File: WebApp/Models/ME/ME_AB0071.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class ME_AB0071 : JCLib.Mvc.BaseModel
    {
        public string CHARTNO{ get; set; }//病歷號
        public string MEDNO{ get; set; }//病歷號碼
        public string VISITSEQ{ get; set; }//門/住診 
        public string ORDERNO{ get; set; }//醫令序號
        public string DETAILNO{ get; set; }//明細序號
        public string ORDERCODE{ get; set; }//院內碼
        public string USEQTY{ get; set; }//數量
        public string CHINNAME{ get; set; }//開立醫師
        public string SIGNOPID{ get; set; }//給藥人員
        public string CREATEDATETIME{ get; set; }//日期
        public string STOCKCODE{ get; set; }//庫房代碼
        public string INOUTFLAG{ get; set; }//A:買 D:退
        public string ORDERENGNAME{ get; set; }//英文名
        public string RESTRICTCODE{ get; set; }//管制用藥
        public string HIGHPRICEFLAG{ get; set; }//高價 
        public string NRCODE{ get; set; }//病房 
        public string BEDNO{ get; set; }//床位 
        public string NRCODENAME{ get; set; }//病房-床位
        public string DOSE{ get; set; }//劑量
        public string ORDERUNIT{ get; set; }//劑量單位
        public string PATHNO{ get; set; }//途徑
        public string FREQNO{ get; set; }//頻率
        public string PAYFLAG{ get; set; }//自費
        public string BUYFLAG{ get; set; }//自備
        public string BAGSEQNO{ get; set; }//藥袋號
        public string RXNO{ get; set; }//處方箋
    }
}

File: WebApp/Scripts/app/model/ME_AB0071.js

Ext.define('WEBAPP.model.ME_AB0071', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'CHARTNO', type: 'string' },//病歷號
        { name: 'MEDNO', type: 'string' },//病歷號碼
        { name: 'VISITSEQ', type: 'string' },//門/住診 
        { name: 'ORDERNO', type: 'string' },//醫令序號
        { name: 'DETAILNO', type: 'string' },//明細序號
        { name: 'ORDERCODE', type: 'string' },//院內碼
        { name: 'USEQTY', type: 'string' },//數量
        { name: 'CHINNAME', type: 'string' },//開立醫師
        { name: 'SIGNOPID', type: 'string' },//給藥人員
        { name: 'CREATEDATETIME', type: 'string' },//日期
        { name: 'STOCKCODE', type: 'string' },//庫房代碼
        { name: 'INOUTFLAG', type: 'string' },//A:買 D:退
        { name: 'ORDERENGNAME', type: 'string' },//英文名
        { name: 'RESTRICTCODE', type: 'string' },//管制用藥
        { name: 'HIGHPRICEFLAG', type: 'string' },//高價 
        { name: 'NRCODE', type: 'string' },//病房 
        { name: 'BEDNO', type: 'string' },//床位 
        { name: 'NRCODENAME', type: 'string' },//病房-床位
        { name: 'DOSE', type: 'string' },//劑量
        { name: 'ORDERUNIT', type: 'string' },//劑量單位
        { name: 'PATHNO', type: 'string' },//途徑
        { name: 'FREQNO', type: 'string' },//頻率
        { name: 'PAYFLAG', type: 'string' },//自費
        { name: 'BUYFLAG', type: 'string' },//自備
        { name: 'BAGSEQNO', type: 'string' },//藥袋號
        { name: 'RXNO', type: 'string' },//處方箋

    ]
});